src/sentry/integrations/slack/message_builder/notifications/issues.py

```diff
@@ -23,6 +24,8 @@ def __init__(
 
     def build(self) -> SlackBlock:
         group = getattr(self.notification, "group", None)
+        assert isinstance(group, Group), "Group must exist to send an issue notification"
```

Comment:
I think (?) maybe the `notification:` annotation in `__init__` could be adjusted to be the specific type that always has `Group` ?